wiki/People/Beatmap_Nominators/Rules/en.md

```diff
@@ -8,46 +8,62 @@ Beatmap Nominators possess the tools to nominate beatmaps and shape the world of
-+ **Disqualifications should primarily be done for beatmaps with unrankable issues, or if the mapset owner requests it.**
-+ **Qualified beatmaps found with unrankable issues must first be posted to the [Report a Beatmap](https://osu.ppy.sh/community/forums/topics/447428) thread.** If the issue has already been reported by someone else, then you are free to disqualify the beatmap. If there is less than 48 hours until rank, it may be disqualified immediately. Getting a second person to confirm greatly reduces the chance that disqualifications are mistakes and is often quick to do.
-+ **Disqualification requests by the mapset owner can be applied immediately.** This includes yourself, in case you want to self-dq.
-+ **For non-unrankable issues or concerns, the mapper should be given time until 24 hours before estimated rank time to respond.** If the mapper fails to respond, the beatmap may be disqualified for further discussion. This does not apply to posts made less than 48 hours before the estimated rank date of the beatmapset, to give mappers ample time to reply and decide whether to make changes.
-+ **When resetting a nomination, the first line of the post must contain a short reason for the reset.** Simply linking to posts is not enough, you should try to summarise the relevant posts. In the case of disqualifications, simply writing that it was requested is also not enough. You must link any related issues posted in the beatmap discussions unless there are too many to post. 
-+ **Ensure that you are looking at the most up to date version of the beatmap before disqualifying.** 
-+ **Beatmaps qualified in modding v1 that need to be disqualified should be brought to the attention of the NAT.** 
-+ **Issues that can be mitigated through online offsets, tags or similar must still be disqualified.** In the case that this is not possible, contact a member of the NAT. Online offsets and tags only apply for the website, so players will still have the issues in the client unless fixed before ranked.
-+ **Only full members may disqualify beatmaps, regardless of reason.** Probation members may however still participate in such discussions and report beatmaps like everyone else.
-
++ **Do not qualify beatmapsets whose song already has a Qualified beatmapset, unless no game modes of both sets overlap - this includes remixed versions**. For example an osu!taiko and osu! beatmapset of the same song can exist in the Qualified section, whilst an osu!taiko and osu!/osu!taiko hybrid set cannot. This ensures variety among Ranked beatmaps within their respective game modes.
++ **Make sure no mod posts in the beatmap thread have been ignored before nominating a beatmap.** Posts without a sufficient reply may count as ignored. When in doubt, double check if the Code of Conduct for Mapping and Modding has been violated on the beatmap thread and act accordingly.
```

Comment:
```suggestion
+ **Make sure no mod posts in the beatmap discussion have been ignored before nominating a beatmap.** Posts without a sufficient reply may count as ignored. When in doubt, double check if the Code of Conduct for Mapping and Modding has been violated on the beatmap thread and act accordingly.
```
thread is used for forums, while modding discussions are called uhhhhhdiscussions